formats/iso0.go

```diff
@@ -2,11 +2,15 @@ package formats
 
 import (
 	"fmt"
+	"io"
 	"strings"
+	"text/tabwriter"
 )
 
 type ISO0 struct {
 	Filler string
+
+	writer io.Writer
```

Comment:
it should be `debugOutput` or `debugWriter` as now meaning of the `writer` is unclear